reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -652,9 +594,8 @@ sudo apt-get install libunwind8 libicu55
 wget http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
 sudo dpkg -i libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
 
-# Download & Install PowerShell
-wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0/powershell_6.0.0-1.ubuntu.16.04_amd64.deb
-sudo dpkg -i powershell_6.0.0-1.ubuntu.16.04_amd64.deb
+# Install PowerShell
+dpkg -i powershell_6.0.2-1.ubuntu.16.04_amd64.deb
```

Comment:
sudo not required?

Reply:
Not sure what happened here but I will fix.